assets/agenda/utils.ts

```diff
@@ -984,75 +984,98 @@ export function isItemTBC(item: IAgendaItem): boolean {
  * @return {String}
  */
 export function formatCoverageDate(coverage: ICoverage) {
+    if (!coverage.scheduled) {
+        return TO_BE_CONFIRMED_TEXT;
```

Comment:
If the Coverage does't have a schedule defined, we shouldn't show anything.
For STT, they don't want to advertise coverage due date at all